src/jit/codegencommon.cpp

```diff
@@ -10317,10 +10333,23 @@ void CodeGen::genFuncletProlog(BasicBlock* block)
 
     compiler->unwindBegProlog();
 
+    inst_RV(INS_push, REG_FPBASE, TYP_REF);
+    compiler->unwindPush(REG_FPBASE);
+
     // TODO Save callee-saved registers
 
     // This is the end of the OS-reported prolog for purposes of unwinding
     compiler->unwindEndProlog();
+
+    // Restore frame pointer
+    // TODO We will need changes here when we introduce PSPSym
+    inst_RV_RV(INS_mov, REG_FPBASE, REG_ARG_0);
```

Comment:
The code looks ok. Can you please set a breakpoint at the `call    edx` in the CallEHFunclet and when you hit is, see if the `bt` command shows a correct stack trace? 
If that doesn't work, the only thing that I can see as possibly needed in addition to the change would be adding `.cfi_def_cfa_offset 4` as the first instruction of the CallEHFunclet